index.html

```diff
@@ -23,7 +23,9 @@
     <div>
        <span>City Name</span> : <span id="current-city">Nandura</span>|
        <br>
-       <span>Temperature Now</span>: <span id="current-temperature">12</span> <span>&deg;C</span>
+       <span>Temperature Now</span>: <span id="current-temperature">NA</span> <span>&deg;C</span>
+       <br>
+       <span>Wind speed</span>: <span id="current-wind-speed">NA </span><span>:m/s</span>
```

Comment:
Thanks for adding NA on load. Since keeping a random number might give user a wrong idea of current weather. This NA shows that the temperature and wind speed are not yet ready to be displayed.